website/docs/language/settings/backends/index.html.md

```diff
@@ -42,9 +42,9 @@ conflicts and inconsistencies.
 
 -> **Note:** In Terraform versions prior to 1.1.0, backends were also classified as being 'standard'
 or 'enhanced', where the latter term referred to the ability of the
-[remote](/docs/language/settings/sbackends/remote.html) backend to store state and perform
+[remote](/docs/language/settings/backends/remote.html) backend to store state and perform
```

Comment:
Just the word "remote" feels like an odd link caption; might we instead make the link text be "the remote backend" so that it covers the full noun phrase?

Reply:
i don't like "the" BUT fair point - I fixe it so it's not just "remote". I agree....that's weird.